diff --git a/node_modules/next/dist/next-server/lib/utils.d.ts b/node_modules/next/dist/next-server/lib/utils.d.ts
index 7c792bb..bc3e556 100644
--- a/node_modules/next/dist/next-server/lib/utils.d.ts
+++ b/node_modules/next/dist/next-server/lib/utils.d.ts
@@ -175,10 +175,23 @@ export declare type DocumentProps = DocumentInitialProps & {
     locale?: string;
     disableOptimizedLoading?: boolean;
 };
+
+
+/**
+ * We don't need a full set of methods since we're only using these two
+ */
+export enum HttpMethod {
+    Post = "POST",
+    Get = "GET"
+}
+
 /**
  * Next `API` route request
  */
 export interface NextApiRequest extends IncomingMessage {
+    // Patching node's method property to not be _any_ string but instead a finite list of known methods.
+    method: HttpMethod;
+
     /**
      * Object of `query` values from url
      */
diff --git a/node_modules/next/types/index.d.ts b/node_modules/next/types/index.d.ts
index 476d491..1a76733 100644
--- a/node_modules/next/types/index.d.ts
+++ b/node_modules/next/types/index.d.ts
@@ -12,6 +12,7 @@ import {
   NextApiResponse,
   NextApiRequest,
   NextApiHandler,
+  HttpMethod,
   // @ts-ignore This path is generated at build time and conflicts otherwise
 } from '../dist/next-server/lib/utils'
 
@@ -88,6 +89,7 @@ export {
   NextApiResponse,
   NextApiRequest,
   NextApiHandler,
+  HttpMethod,
 }
 
 export type PreviewData = string | false | object | undefined
